### ISSUE TITLE
Add Other Learning Resources

### ISSUE THREAD

**Issue by oren** [elibenporat/learn-to-code-rust-baseball#7]
I found another good Rust tutorial that reminds me yours:
https://news.ycombinator.com/item?id=23206660

**Comment by elibenporat** [comment]
This is really good! If you come across other good resources, add them here. There are a bunch being done. 

If you like live streams, Jon Gjengset does a fantastic job of them: https://www.youtube.com/channel/UC_iD0xppBwwsrM9DegC5cQQ They're on the more advanced side, but that shouldn't be a problem for you given your background.

Going to change this to "Add Other Learning Resources".

**Comment by oren** [comment]
This is going to be amazing. Look at the table of contents: it's features of the product and not language features.

https://www.reddit.com/r/rust/comments/gpydni/zero_to_production_in_rust_0_foreword/

**Comment by elibenporat** [comment]
Agree, this looks very promising. Best feature of Rust is the community.

**Comment by oren** [comment]
This is another great tutorial: https://kbknapp.dev/rust-cli/ (it was mentioned in 'This Week in Rust 344'). I just spent 5 minutes on it and I already love it. 

The first thing I did is cloned the repo (which is mentioned at the end) and played with the final product. Now that I know what I am building I am going through the tutorial itself. I also love the Overview section where you see the structure of the program (the main function, and the 4 structs). I also like that the app has 3 files - it shows me how to split an app to smaller chunks which is important as my code grows beyond hello world.
